corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -156,6 +156,12 @@
                type="checkbox"
                data-bind="value: uniqueId, checked: $parent.includedFormIds"/>
         <span data-bind="text: name"></span>
+        {% if session_endpoints_enabled %}
+          <label>
+            <span>Session Endpoint Id:</span>
```

Comment:
translate, and no need for a colon

Reply:
you might also consider doing this text as a placeholder to reduce the text on the screen

Reply:
I am not sure what you mean with placeholder?